service/policy/objects.proto

```diff
@@ -265,16 +265,38 @@ message KeyAccessServer {
   common.Metadata metadata = 100;
 }
 
+// A KAS public key and some associated metadata for further identifcation
```

Comment:
Since we [only support two keys](https://github.com/opentdf/platform/blob/main/service/internal/security/crypto_provider.go#L8-L13) we should prevent users for uploading unsupported keys

```suggestion
enum KasPublicKeyAlgEnum {
  KAS_PUBLIC_KEY_ALG_ENUM_UNSPECIFIED = 0;
  KAS_PUBLIC_KEY_ALG_ENUM_RSA_2048 = 1;
  reserved 2 to 4;
  KAS_PUBLIC_KEY_ALG_ENUM_EC_SECP256R1 = 5;
  reserved 5 to 100;
}

// A KAS public key and some associated metadata for further identifcation
```